.circleci/config.yaml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  deploy:
+    docker:
+      # Upgrade the Dockerfile and push image when the hub az cluster tiller
+      # version upgrades.
+      - image: r6by/kubectl-helm-azure
+        # Don't automatically upgrade the Hub monocular chart version for each
+        # new helm/monocular release. Instead assume we'll need testing, and
+        # set the version manually here.
+        environment:
+          CHART_VERSION: 1.4.4
+    steps:
+      - run:
+          name: Authenticate to cluster
+          command: |
+            az login --service-principal -u $SERVICE_PRINCIPAL_ID -p $SERVICE_PRINCIPAL_PASS -t $SERVICE_TENANT_ID
+            az aks get-credentials -n $CLUSTER_NAME -g $RESOURCE_GROUP
+      - checkout
+      - run:
+          name: Upgrade/install Monocular Helm chart with any Hub changed values
+          command: |
+            export MONGODB_ROOT_PASSWORD=$(kubectl get secret --namespace helm-hub helm-hub-mongodb -o jsonpath="{.data.mongodb-root-password}" | base64 -d)
+            helm init --client-only
+            helm repo add monocular https://helm.github.io/monocular
+            helm repo update
+            helm upgrade --install helm-hub monocular/monocular --version $CHART_VERSION --namespace helm-hub --set mongodb.mongodbRootPassword=$MONGODB_ROOT_PASSWORD -f config/helm-hub-values.yaml -f config/repo-values.yaml
+
+workflows:
+  version: 2
+  merge-master:
+    jobs:
+      - deploy:
+          filters:
+            branches:
+              only: master
```

Comment:
This is going to run on every change to the repo including docs updates and things unrelated to the config. Can we have a script check for just config changes that should be deployed?

Reply:
Agree, just for any change in one of the three YAML files 👍

Reply:
This should now be checking changes to only `config/helm-hub-values.yaml`, `config/repo-values.yaml` and `repos.yaml`.

Reply:
- [x] Tested on my fork